lib/util/move-file.js

```diff
@@ -23,6 +23,7 @@ function moveFile (src, dest) {
   return new Promise((resolve, reject) => {
     fs.link(src, dest, (err) => {
       if (err) {
+        /* istanbul ignore else */
```

Comment:
I think this should be testable with http://npm.im/mutate-fs

Reply:
oh, almost all the lines here already have coverage! the only missing line is **L29** since it's windows specific, maybe I should move the comment to the same line 🤔

Reply:
ok that should be much better now, I moved it to stay in the same line of that platform-dependent condition and it only ignores that single statement instead of all `else` blocks 😊